crates/re_data_source/src/lib.rs

```diff
@@ -43,13 +43,16 @@ pub const SUPPORTED_IMAGE_EXTENSIONS: &[&str] = &[
 
 pub const SUPPORTED_MESH_EXTENSIONS: &[&str] = &["glb", "gltf", "obj"];
 
+pub const SUPPORTED_POINT_CLOUD_EXTENSIONS: &[&str] = &["ply"];
```

Comment:
this deserves a comment, since `.ply` is both a mesh and a point cloud format (as you pointed out in the PR descr)